### PR TITLE
Help message

### PR DESCRIPTION
**Pull request by 16aburnett**

### CODE REVIEW

File: code/UMLEditor.py [hunk 1]
```diff
@@ -14,6 +14,29 @@
...
+        }
+    ],
+    "create_class" : [
+        {
+            "usage" : "create_class <class_name>",
+            "desc" : "creates a class with <class_name> as the name"
+        }
+    ]
+}
+
```
Comment (16aburnett) [inline code/UMLEditor.py:38]: As a separate task, we can go through and add in all the commands, usage statements, and descriptions. This was just a sample to test that the function works

**Comment by 16aburnett** [comment]
Should we move the command usages and descriptions into its own file? It really clutters the UMLEditor file

**Review by carlishires — CHANGES_REQUESTED** [review]
Typing "help" in the command line while UMLEditor does only outputs "To be implemented".

**Comment by 16aburnett** [comment]
> Typing "help" in the command line while UMLEditor does only outputs "To be implemented".

The "To be implemented" comes from the execute function which is what David and Adisa were working on.

**Comment by carlishires** [comment]
> > Typing "help" in the command line while UMLEditor does only outputs "To be implemented".
> 
> The "To be implemented" comes from the execute function which is what David and Adisa were working on.

Alright, thanks for telling me.

**Review by carlishires — APPROVED** [review]
print_help_message appears to be in working order

**Pull request merged**